src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/util/WhiteListSingleton.java

```diff
@@ -0,0 +1,55 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.util;
+
+import de.uni_mannheim.informatik.dws.winter.utils.WinterLogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.HashSet;
+import java.util.Set;
+
+public class WhiteListSingleton {
+
+    public static final Logger logger = WinterLogManager.activateLogger("default");
```

Comment:
You can reuse the global logger with `IR_App.logger`. No need to create a new one.